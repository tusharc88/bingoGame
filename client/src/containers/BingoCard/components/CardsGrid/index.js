import React from 'react';
import styled from 'styled-components';
import Grid from '../../../../components/Grid/index';

const CardsGridWrapper = styled.div`
  display: flex;
  justify-content: space-around;
  flex-wrap: wrap;
  width: 70%;
`;

const CardsGrid = () => (
  <CardsGridWrapper>
    {Array.from({ length: 4 }, (v, k) => <Grid matrixNum={5} key={k} />)}
  </CardsGridWrapper>
);

export default CardsGrid;
