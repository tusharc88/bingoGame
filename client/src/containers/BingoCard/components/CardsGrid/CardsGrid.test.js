import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import CardsGrid from './index';

test('Renders with default props', () => {
  const tree = renderer.create(<CardsGrid />).toJSON();
  expect(tree).toMatchSnapshot();
});
