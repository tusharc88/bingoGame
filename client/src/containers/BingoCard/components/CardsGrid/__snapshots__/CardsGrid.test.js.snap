// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders with default props 1`] = `
.c3 {
  height: 40px;
  width: 40px;
  background-color: #d7d3d2;
  color: #393838;
  font-size: 25px;
  text-align: center;
  vertical-align: middle;
  line-height: 40px;
  border: 1px solid #ffffff;
  -webkit-transition: all 0.3s ease-in-out;
  transition: all 0.3s ease-in-out;
  -webkit-transition-property: box-shadow,color,background-color;
  transition-property: box-shadow,color,background-color;
  cursor: pointer;
}

.c3:hover {
  box-shadow: inset 0 0 0 3px #3ecedf;
  -webkit-transition: 'all 0.3s ease-in-out';
  transition: 'all 0.3s ease-in-out';
  -webkit-transition-property: 'box-shadow, color, background-color';
  transition-property: 'box-shadow, color, background-color';
}

.c1 {
  -webkit-transition: all 0.3s ease-in-out;
  transition: all 0.3s ease-in-out;
  -webkit-transition-property: -webkit-transform box-shadow;
  -webkit-transition-property: transform box-shadow;
  transition-property: transform box-shadow;
}

.c1:hover {
  box-shadow: 0px 5px 5px 0px #aeb0b1;
  -webkit-transform: translateY(-3px);
  -ms-transform: translateY(-3px);
  transform: translateY(-3px);
  -webkit-transition: all 0.3s ease-in-out;
  transition: all 0.3s ease-in-out;
  -webkit-transition-property: -webkit-transform box-shadow;
  -webkit-transition-property: transform box-shadow;
  transition-property: transform box-shadow;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  width: 70%;
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    <div
      className="c2"
    >
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
    </div>
    <div
      className="c2"
    >
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
    </div>
    <div
      className="c2"
    >
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
    </div>
    <div
      className="c2"
    >
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
    </div>
    <div
      className="c2"
    >
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
    </div>
  </div>
  <div
    className="c1"
  >
    <div
      className="c2"
    >
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
    </div>
    <div
      className="c2"
    >
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
    </div>
    <div
      className="c2"
    >
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
    </div>
    <div
      className="c2"
    >
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
    </div>
    <div
      className="c2"
    >
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
    </div>
  </div>
  <div
    className="c1"
  >
    <div
      className="c2"
    >
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
    </div>
    <div
      className="c2"
    >
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
    </div>
    <div
      className="c2"
    >
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
    </div>
    <div
      className="c2"
    >
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
    </div>
    <div
      className="c2"
    >
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
    </div>
  </div>
  <div
    className="c1"
  >
    <div
      className="c2"
    >
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
    </div>
    <div
      className="c2"
    >
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
    </div>
    <div
      className="c2"
    >
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
    </div>
    <div
      className="c2"
    >
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
    </div>
    <div
      className="c2"
    >
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
      <div
        className="c3"
        disabled={false}
        onClick={[Function]}
      >
        1
      </div>
    </div>
  </div>
</div>
`;
